src/pfe/portal/modules/LoadRunner.js

```diff
@@ -523,22 +520,22 @@ module.exports = class LoadRunner {
    * The samples are written to a file as a JSON array of samples.
    */
   async endProfiling() {
-
     if (this.profilingSocket !== null) {
       await fs.writeJson(this.workingDir + '/profiling.json', this.profilingSamples, { spaces: '  ' }, function (err) {
         if (err) {
           log.error('endProfiling: Error writing profiling samples');
           log.error(err);
         }
       });
+      log.info(`profiling.json saved for project ${this.project.name}`);
       const data = { projectID: this.project.projectID,  status: 'profilingReady' , timestamp: this.metricsFolder }
       this.user.uiSocket.emit('runloadStatusChanged', data);
       this.profilingSocket.emit('disableprofiling');
       this.profilingSocket.disconnect();
       this.profilingSocket = null;
       this.profilingSamples = null;
+      this.project = null;
     }
-    this.project = null;
```

Comment:
We don't null the project if there wasn't a socket? Can we run load against projects we don't create sockets for? (It looks like we only create sockets for Node projects.)